<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminpage</class>
 <widget class="QMainWindow" name="Adminpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;
background-color:floralwhite;
border-radius:5px;
font: 12pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:floralwhite;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
border: 1px solid green;
color: white;
border-collapse:collapse;
background-color: green;</string>
     </property>
     <property name="text">
      <string>View Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
border: 1px solid green;
color: white;
border-collapse:collapse;
background-color: green;</string>
     </property>
     <property name="text">
      <string>Edit Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
border: 1px solid green;
color: white;
border-collapse:collapse;
background-color: green;</string>
     </property>
     <property name="text">
      <string>New Employee</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
background-color:white;
color:black;
border-radius:2px;</string>
     </property>
     <property name="placeholderText">
      <string>enter amount</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
border: 1px solid green;
color: white;
border-collapse:collapse;
background-color: green;</string>
     </property>
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>9</y>
      <width>331</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: rgb(51, 51, 51);
border-radius:5px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="profile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>311</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>601</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:floralwhite;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
border: 1px solid beige;
color: black;
border-collapse:collapse;
background-color: rgb(244, 228, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search employee username:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
border: 1px solid blue;
color: white;
border-collapse:collapse;
background-color: blue;</string>
     </property>
     <property name="text">
      <string>search</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="searchresulttable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:floralwhite;
color:black;
font: 10pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="expiremessage">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bookbutton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border: 1px solid blue;
color: white;
border-collapse:collapse;
background-color: blue;</string>
    </property>
    <property name="text">
     <string>Create shop book</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
background-color:white;
color:black;
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
